src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerMaintainLinkCase.java

```diff
@@ -36,7 +34,7 @@ public class CaseWorkerMaintainLinkCase implements CCDConfig<CaseData, State, Us
     @Value("${feature.link-case.enabled}")
     private boolean linkCaseEnabled;
 
-    private static final CcdPageConfiguration linkCaseSelectCase = new LinkCaseSelectCase();
+    //private static final CcdPageConfiguration linkCaseSelectCase = new LinkCaseSelectCase();
```

Comment:
Doesn't seem to be related to case flag feature. If not, revert back.

Reply:
Reverted the change.